civihr_default_theme/scss/drupal/_form.scss

```diff
@@ -44,23 +44,39 @@
 }
 
 #views-exposed-form-users-list-page {
-  .views-exposed-widget:not(:last-child) {
+  .views-exposed-widget {
     float: left;
     width: 50%;
+  }
 
-    &:nth-child(odd) {
-      padding-right: 20px;
-    }
+  .views-submit-button {
+    float: none;
+    width: auto;
+  }
+
+  .views-widget-filter-name,
+  .views-widget-filter-rid,
+  .views-widget-filter-access {
+    padding-right: 20px;
+  }
+
+  .views-widget-filter-access,
```

Comment:
what are all the new styles? The description doesn't mention them

Reply:
Explained in the description.